liquibase-standard/src/main/java/liquibase/util/SqlUtil.java

```diff
@@ -114,6 +112,12 @@ public static Object parseValue(Database database, Object val, DataType type) {
             }
         }
 
+        if ((database instanceof PostgresDatabase) && (liquibaseDataType instanceof CharType || liquibaseDataType instanceof ClobType)
```

Comment:
I think we can add Oracle to the database check of this condition:

```(database instanceof PostgresDatabase || database instanceof OracleDatabase)```  &&...

Reply:
Ok, for the future. But, for now, we will never be here because of the conditions above:

```java
else if (!stringVal.matches("\\d+\\.?\\d*")) {
                //not just a number
                return new DatabaseFunction(stringVal);
```

Reply:
Added